src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/components/MigrateProjectController.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import { InfoCallout } from 'components/Callout/InfoCallout'
+import EtherscanLink from 'components/EtherscanLink'
+
+/**
+ * Component to call `migrateController` on a project.
+ */
+export function MigrateProjectController({
```

Comment:
This is a 'generic' component we can use for future JBController migrations (like 3.1)